css/style.css

```diff
@@ -98,3115 +98,545 @@
 /* ----------------------------[ Fonts ]---------------------------------- */
 
 @font-face {
-  font-family: FontAwesome;
-  font-weight: normal;
-  font-style: normal;
-  src: url("../fonts/fontawesome.woff2");
+	font-family: FontAwesome;
+	font-weight: normal;
```

Comment:
`un-prettyify`...? :p Heh, that's quite ironic. The tabs were already pretty, 2-spaces made it the opposite. 😁